chuck_norris_facts.rb

```diff
@@ -0,0 +1,15 @@
+require 'faker'
+
+flag = true
+
+puts 'True fact about Chuck Norris'
+
+while flag
+    puts Faker::ChuckNorris.fact
+    puts 'Do you want more? Y or N'
+    if gets.chomp.upcase == 'N'
+        puts "Bye!"
+        flag = false
+        break
```

Comment:
Nit: There's no need for both a flag and `break`.

Reply:
Done